src/export/utils/localizationOptions.ts

```diff
@@ -11,8 +12,7 @@ export function localizationOptions(
   return {
     localizeImageSrc: (src) => imageFilenames[src],
     localizeId: (id) => id.split('#')[1], // TODO: this is a hack
-    // TODO: needs to be expanded to look up
-    localizeCitation: (key) => references[key].label,
+    localizeCitation: (key) => references[basekey(key)].label,
```

Comment:
@rowanc1 started out doing a lookup here, but then thought as we have control over references this would be better